seeders/20180226193120-demo-user.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
```

Comment:
Remove this demo migration, its fine to test things in local but it shouldn't be commited